src/utils/defaultLanguageStrings.ts

```diff
@@ -148,7 +148,7 @@ export default {
   FILTER_DATE_BEFORE: 'Before',
   FILTER_DATE_AFTER: 'After',
   FILTER_RESET_ALL: 'Reset all filters',
-  LABEL_APPLICATIONS: 'applications',
+  LABEL_APPLICATIONS: '%1 applications',
   LABEL_APPLICATION_SUBMITTED: 'Application Submitted',
```

Comment:
I'm so glad to see this for pluralisation, makes so much more sense and is easier to read/more consistent. Does that mean we can finally ditch the unnecessary (and largely unused) "name_plural" column?

Reply:
No. It's still used in the Dashboard. As mentioned previously Plural in other languages aren't as simple as English

Reply:
Okay, I'll have to get you to explain it to me again sometime, cos I don't quite get it. In English, it's just nicer to say "Org Registration applications" rather than "Org Registrations" (so you don't need a specific plural), but I accept that might not translate so well.